junit-plugin/src/main/scala/org/junit/RuntimeStubs.scala

```diff
@@ -0,0 +1,12 @@
+package org.junit
+
+/* Stubs
+ * Defined in `junit-runtime`
+ */
+class Test
```

Comment:
Oh. Yes, there is. Just don't use `typeOf[org.junit.Test]`. Use `getRequiredClass("org.junit.Test").toTypeConstructor`. Or even better in some cases, just stick to the `Symbol` by dropping `.toTypeConstructor`.